mapbox/dr-ui
# Improve color contrast on code and anchor elements

**This branch merges into #431**

This PR adds a few CSS rules to improve color contrast on elements. 

Fixes #426 
Fixes #425 
Fixes #424 

## How to test

1. `npm run start-docs`
2. Visit http://localhost:8080/dr-ui/guides/a11y/
3. Run Lighthouse's Accessibility audit, the score should be 100%.

## QA checklist



- [x] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `7.0.0`.

## Before merge
